Essentials/src/main/java/com/earth2me/essentials/Settings.java

```diff
@@ -626,7 +631,7 @@ public void reloadConfig() {
         signUsePerSecond = _getSignUsePerSecond();
         chatFormats.clear();
         changeDisplayName = _changeDisplayName();
-        disabledCommands = getDisabledCommands();
+        disabledCommands = _getDisabledCommands();
```

Comment:
Either at the start of the method or before this line, we should be re-adding commands that were removed from this list back to the known commands map. Maybe store the values we remove from the map in some other place and just compare that with the current disabled commands list?

Reply:
I'm just re-adding all removed commands for now, since that's a much simpler solution. If they're still in the disabled commands list after a reload, they will be re-removed.